src/client.js

```diff
@@ -480,47 +503,60 @@ function SugarApi(args) {
         };
     };
 
-
     return {
         /**
-         * Client Id for oAuth
-         * @property {String}
-         * @member Api
+         * Client Id for oAuth.
+         *
+         * @property {string}
```

Comment:
Can you use `@type {string}` instead of `@property {string}`? Likewise for the other properties below.